### ISSUE TITLE
Data was Illicitly Scraped

### ISSUE THREAD

**Issue by LotosAbstract** [moaatt2/chainmail_stats#2]
Full data has been scraped in violation of content policy from mailleartisans.org.

I am in possession of the server logs that show and prove the same.

Asking you to acknowledge this fact, and agree to remove this data, prior to a DMCA request being filed with GitHub.

**Comment by moaatt2** [comment]
Good evening @LotosAbstract,

First, I have removed the files `data.json` and `data.csv` from this repository and repository history so it will not be available. I am very sorry to have violated the terms of mailleartisans.org as I only wanted to learn more about chainmail and practice data analysis. I made the data available in the repository with the hope of giving as much back to the community as possible and so that others could do data analysis without putting a drain on the website.

I have checked the [robots.txt](https://www.mailleartisans.org/robots.txt), [content policy](https://www.mailleartisans.org/help/content_policy.php), and [looked for any additional content policy pages](https://www.google.com/search?q=content+policy+site%3Amailleartisans.org) yet I was unable to find where I made the violation. If you could please  show me on the website what rule I broke I would greatly appreciate it.

I am sorry for any trouble that I have caused and I hope that we can resolve this quickly.

**Comment by moaatt2** [comment]
There appears to be an issue with the deletion of the files from the repo history. I am working on that now.

**Comment by LotosAbstract** [comment]
Thank you for your prompt attention to this!

Unfortunately, all submitted data to maillartisans.org remains property of the original submitter. (See Footer on each page.)
As we hold the data, we also have a duty to safeguard this arrangement.

Regretfully, maillartisans.org is not an open-source data source, although there has been desire amongst some to move in that direction in the past.

Again, thank you for handling this removal!

**Comment by moaatt2** [comment]
Firstly and most importantly, I have taken the appropriate steps to remove as much information as possible from the repository and have reached out to GitHub support for assistance in dealing with the remaining data. 

Additionally, is there any of the already publicly available information that can be open sourced(with appropriate attribution), such as `Weave Name` and `AR` (like the data available in Joshua Dilberto's [Tome of Weaves](https://www.joshuadiliberto.com/JD_supportFiles/weaves_A.php)). Opening this information would be helpful in many ways, one that I can think of immediately is streamlining the validation of new weave names. Currently, this involves searching 30 individual pages on M.A.I.L. which is error prone and time consuming.

Moreover, I've heard from others that the forums contain valuable information for newcomers in the field, like myself. Unfortunately, they are currently inaccessible. If there is a way to make them accessible(perhaps a downloadable static offline website) I would love to help if possible.

Finally, I am interested in establishing a line of communication with you for future data analysis/personal research purposes. Please let me know if this works for you.

In summary, I appreciate mailleartisans.org as an invaluable repository of information. However, certain functionalities and data is no longer accessible. Many, including myself, would greatly benefit from having access to and learning from this information.

**Comment by moaatt2** [comment]
@LotosAbstract I have cleared all the data from the repository. Please confirm that this resolution is accepted and I will close this issue.

**Comment by moaatt2** [comment]
@LotosAbstract, it has been a week since I requested confirmation that the current situation is acceptable. I will be marking this issue closed and resolved unless I hear back from you before December 16th 2023.

**Comment by moaatt2** [comment]
@LotosAbstract as there has been no feedback I will consider this issue resolved.
